src/mocks/MockERC20.sol

```diff
@@ -1,36 +1,54 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.6.2 <0.9.0;
 
+import {IERC20} from "../interfaces/IERC20.sol";
+
 /// @notice This is a mock contract of the ERC20 standard for testing purposes only, it SHOULD NOT be used in production.
 /// @dev Forked from: https://github.com/transmissions11/solmate/blob/0384dbaaa4fcb5715738a9254a7c0a4cb62cf458/src/tokens/ERC20.sol
-contract MockERC20 {
+contract MockERC20 is IERC20 {
     /*//////////////////////////////////////////////////////////////
-                                 EVENTS
+                            METADATA STORAGE
     //////////////////////////////////////////////////////////////*/
 
-    event Transfer(address indexed from, address indexed to, uint256 amount);
+    string private _name;
```

Comment:
Let's change all `private` vars to `internal` so devs have more control when inheriting from this contract and making custom mock tokens

Reply:
Agree